std/http/file_server_test.ts

```diff
@@ -166,3 +180,14 @@ test("contentType", async () => {
   assertEquals(contentType, "text/html");
   (response.body as Deno.File).close();
 });
+
+test("file_server running as library", async function (): Promise<void> {
+  await startFileServerAsLibrary();
+  try {
+    const res = await fetch("http://localhost:4507");
```

Comment:
Try using `127.0.0.1`

Reply:
I already tried that but without success :(